api/social_nets/interfaces.py

```diff
@@ -0,0 +1,32 @@
+from abc import ABCMeta, abstractmethod, abstractproperty
+
+from typing import Union
+
+
+class ISocialNet:
+    __metaclass__ = ABCMeta
+
+    @abstractmethod
+    def call(self, method: str, params: dict) -> dict:
+        """Call API method"""
+
+    @abstractmethod
+    def send_message(self, chat_id: Union[str, int], text: str) -> dict:
+        """Send message to chat"""
+
+    @abstractmethod
+    def send(self, text: str, chat_id: Union[str, int]) -> None:
+        """Safety send message to chat"""
+
+    @property
+    @abstractmethod
+    def api_url(self):
```

Comment:
Maybe use @abstractproperty?

Reply:
> Maybe use @abstractproperty?

No, because @abstractproperty now is deprecated and we should use @property with @abstractmethod